clients/src/main/java/org/apache/kafka/common/internals/InvalidProducerEpochException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Why not put this internal exception in `org.apache.kafka.clients.producer.internals`?

Reply:
sg